source/slang/slang-emit.cpp

```diff
@@ -542,6 +542,10 @@ Result linkAndOptimizeIR(
 
     validateIRModuleIfEnabled(codeGenContext, irModule);
 
+    // Fold all calls to builtin functions that results in simple integer values.
+    // When postInlining flag is set, this happens for alignOf<T>() function, which
+    // infers the size of T and replaces the use with the IntLiteral.
+    peepholeOptimize(targetProgram, irModule, PeepholeOptimizationOptions::getPostInlining());
```

Comment:
I don't think this is needed because we just run it inside the simplifyIR call above.

Reply:
got it, i updated the peephole optimize call within SimplyIR with the getPostInlining() flag set.